source/api/presentation/3.0/index.md

```diff
@@ -1679,7 +1679,7 @@ Clients _MUST NOT_ attempt to construct resource URIs by themselves, instead the
 
 ###  6.3. Responses
 
-The format for all responses is JSON, as described above. The different requirements for which resources _MUST_ provide a response is summarized in [Appendix A][prezi30-appendixa]. While some resources do not require their URI to provide the description, it is good practice if possible.
+The format for all responses is JSON, as described above. It is good practice for all resources with an HTTP(S) URI to provide their description when the URI is dereferenced. If the resource is referenced, rather than being embedded in its entirety, then it _MUST_ be able to be dereferenced.
```

Comment:
>If _the_ resource is referenced

We're not talking about a specific instance, so I think the article should be _a_, instead of _the_:

> If _a_ resource is referenced 

Also, is 'embedded in its entirety' clear, or should it be something more explicit, like, 'contained with the current resource' or something like that?

Reply:
Agree that could do with a tiny bit more here to indicate that "embedded" is being used in the formal sense defined in https://iiif.io/api/presentation/3.0/#12-terminology

Reply:
This might just be removing "in its entirety" to match other uses of the term elsewhere in the spec.

Reply:
Linked "referenced" and "embedded" to the terminology section and removed "in its entirety". Swapped "the" for "a".